server/routes/pub.js

```diff
@@ -73,13 +73,30 @@ app.get(
 						mode: mode,
 					},
 				};
+				let primaryCollection = false;
+				pubData.collectionPubs.forEach((collection) => {
+					if (collection.isPrimary && collection.collection.kind !== 'issue') {
+						primaryCollection = collection.collection;
+					}
+				});
+				const collectionTitle = primaryCollection.title || initialData.communityData.title;
```

Comment:
In the case where there is no primaryCollection, calling `primaryCollection.title` (i.e. `false.title`) will throw an error. I'll add a commit where we use a reduce instead to handle this case.